test/e2e/flask/multichain-api/create-session.spec.ts

```diff
@@ -46,4 +61,144 @@ describe('Multichain API', function () {
       );
     });
   });
+
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_createSession` with requested EVM scope that match the user’s enabled networks', function () {
+    it('should only select the specified EVM scopes requested by the user, and session scope contain valid accounts only', async function () {
```

Comment:
```suggestion
  describe('Call `wallet_createSession` with EVM scopes that match the user’s enabled networks, and eip155 scoped accounts', function () {
    it('should ignore requested accounts that do not match accounts in the wallet and and pre-select matching requested accounts in the permission confirmation screen', async function () {
```

Reply:
resolved in [2693ebb](https://github.com/MetaMask/metamask-extension/pull/29356/commits/2693ebb31a0fbdddfe91c677402ea8b2e3b061f5)